modules/codemod/lib/v8/spec/promoteBreadcrumbs.spec.ts

```diff
@@ -18,6 +18,17 @@ describe('Rename Box, Flex and Stack', () => {
       expectTransform(input, expected);
     });
 
+    it('should not transform other preview imports', () => {
+      const input = stripIndent`
+          import { Select } from "@workday/canvas-kit-preview-react/select";
+          import { SegmentedControl } from "@workday/canvas-kit-preview-react/segmented-control";
+        `;
+
+      const expected = input;
+
+      expectTransform(input, expected);
+    });
+
```

Comment:
I have another spec to suggest (rather err on the side of not missing something) but it's optional if you add it:

```
    it('should not add Breadcrumbs when it is not already imported', () => {
      const input = stripIndent`
          import { Select } from "@workday/canvas-kit-preview-react";
        `;

      const expected = stripIndent`
      import { Select } from "@workday/canvas-kit-preview-react";
        `;

      expectTransform(input, expected);
    });
```

This looks like the one you have but imports from the main package. I already tested your fix against it and it works all the same!!

Reply:
Thanks for suggestion. It's added